src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotConstantPool.java

```diff
@@ -808,7 +808,7 @@ public JavaField lookupField(int cpi, ResolvedJavaMethod method, int opcode) {
         JavaType fieldHolder = lookupType(holderIndex, opcode);
 
         if (fieldHolder instanceof HotSpotResolvedObjectTypeImpl) {
-            int[] info = new int[3];
+            int[] info = new int[4];
             HotSpotResolvedObjectTypeImpl resolvedHolder;
             try {
                 resolvedHolder = compilerToVM().resolveFieldInPool(this, index, (HotSpotResolvedJavaMethodImpl) method, (byte) opcode, info);
```

Comment:
Please update the javadoc for `CompilerToVM.resolveFieldInPool` to reflect the expanded definition of `info`.